app/models/operatingsystems/redhat.rb

```diff
@@ -72,4 +81,33 @@ def pxe_kernel_options(params)
   def minor_version_help
     '0, 6.1810'
   end
+
+  def is_fedora?
+    name == 'Fedora'
+  end
+
+  def is_rhel?
```

Comment:
You can't say this. CentOS is not RHEL but it certainly is within the Red Hat OS family. That's why we always used `rhel_compatible`.